src/App.js

```diff
@@ -0,0 +1,25 @@
+import {
+  BrowserRouter as Router, Routes,
+  Route,
+} from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Books from './components/Books';
+import Category from './components/Category';
+import Form from './components/Form';
+
+function App() {
+  return (
+    <div>
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/category" element={<Category />} />
+          <Route path="/form" element={<Form />} />
```

Comment:
- [ ] Kindly remove this route as the form is already inside the Books component.